azerothcore/azerothcore-wotlk
# feat(Core/Command): .account onlinelist orders accounts by IP

## CHANGES PROPOSED:
-  .account onlinelist now orders accounts by IP
- There was no order by default, so I added the order by IP because it's quite useful to group accounts by IP, to see multi accounts (notably multiboxers)


## TESTS PERFORMED:
- I can't test it, I have only one IP




## HOW TO TEST THE CHANGES:

- Read the code, and review :P 
- Otherwise, need a server with players online and you connect with two accounts, type `.account onlinelist` and see if your two accounts are grouped together


## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
